test/structured_data_tags_test.js

```diff
@@ -380,7 +380,11 @@ describe('StructuredDataTags', function() {
     it("adds BreadcrumbList schema by default", function() {
       const string = getSeoMetadata(getSeoConfig(), {}, 'section-page', {"data": {"section": { "section-url": "https://madrid.quintype.io/film", "slug": "film", "name": "Film" }}, config: {"sketches-host": "https://madrid.quintype.io", "sections": [{"section-url": "https://madrid.quintype.io/film", "slug": "film", "name": "Film", "parent-id": null}]}}, {url: url.parse("/")});
       assertContains(sampleBreadcrumbListTag, string);
-    })
+    });
+    it("adds BreadcrumbList schema for custom page", function() {
+      const string = getSeoMetadata(getSeoConfig(), {}, 'custom-page', {"data": {"collection": { "pagePath": "https://madrid.quintype.io/film", "slug": "film", "name": "Film" }}, config: {"sketches-host": "https://madrid.quintype.io", "sections": [{"section-url": "https://madrid.quintype.io/film", "slug": "film", "name": "Film", "parent-id": null}]}}, {url: url.parse("/")});
```

Comment:
Instead of collection, can you use breadcrumbs as I mentioned in the other comment?

Reply:
Yeah, modified the test case according to the modified code. Can you please review it?